src/main/java/com/jmb/springfactory/model/entity/Task.java

```diff
@@ -42,10 +39,15 @@
 
   private String reasonRejection;
 
+  private String blockedReason;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "blockedReason" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)